<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="pbSetGlobalShortcut">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Set &amp;Global shortcut</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="4">
      <widget class="QLineEdit" name="leGlobalShortcut">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pbSetLocalShortcut">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Set &amp;Local shortcut</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QLineEdit" name="leLocalShortcut">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pbSetArea">
       <property name="text">
        <string>Set &amp;Area</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QLineEdit" name="leArea">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QToolButton" name="pbRemove">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QToolButton" name="pbAbout">
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="lwShortcuts"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
